webapp/app/forms/flows/lotse_flow.py

```diff
@@ -354,7 +358,7 @@ def _collect_data_for_step(self, step, form_data, missing_fields=None):
                 field = getattr(step.form, attr)
                 label, value = self._generate_value_representation(field, form_data[attr])
                 if value:
-                    step_data[label] = value
+                    step_data[label] = step.get_overview_value_representation(value)
```

Comment:
I do like that we have a specific function here now. However,  was imagining this (meaning  `step.get_overview_value_representation(...)`) would replace `self._generate_value_representation(field, form_data[attr])`? Then, in the most abstract step we could have the exact implementation of `self._generate_value_representation(field, form_data[attr])` and just override it when necessary?

Reply:
Yes I'm with you, then we also don't have a IfElse tree in the lotse_flow anymore. But that change would be big and has a impact on many components... Maybe add it as a technical debt? :eyes:

Reply:
Jap. We already added a comment, right? So I guess that's fine?